Source/WebCore/style/StyleAdjuster.cpp

```diff
@@ -928,6 +928,19 @@ void Adjuster::adjustForSiteSpecificQuirks(RenderStyle& style) const
         if (style.overflowY() == Overflow::Hidden && m_element->attributeWithoutSynchronization(roleAttr) == roleValue)
             style.setOverflowY(Overflow::Auto);
     }
+    if (m_document.quirks().needsIPadSkypeOverflowScrollQuirk()) {
+        // This makes the layout scrollable and makes visible the buttons hidden outside of the viewport.
+        // static MainThreadNeverDestroyed<const AtomString> selectorValue(".app-container .noFocusOutline > div"_s);
+        if (RefPtr div = dynamicDowncast<HTMLDivElement>(m_element)) {
+            auto matchesNoFocus = div->matches(".app-container .noFocusOutline > div"_s);
+            if (matchesNoFocus.hasException())
```

Comment:
This function is gaining numerous separate quirks that require different behavior. Would it better to have separate functions for each website. I see this specific quirk function is handling Skype, Youtube, and WeChat.

Reply:
yes that's a good idea. I think there's a need for a basic function to override CSS in general.